src/main/java/edu/ut/ece/social/hw1/HwRunner.java

```diff
@@ -30,6 +33,28 @@ public static void runMaximumMatchingProblem(String filename, MaximumMatchingAlg
 
     }
 
+    public static String runMaximumMatchingProblemWithOutput(String filename, MaximumMatchingAlgorithm matchingAlgorithm)
+            throws FileNotFoundException {
+        BipartiteGraph<Integer, Integer> graph = InputReader.readGraphFromFile(Paths.get(filename));
+
+        Instant startInstant = Instant.now();
+        Matching<Integer> maxMatching = matchingAlgorithm.findMaximumMatching(graph);
+        Instant endInstant = Instant.now();
+
+        Duration elapsedTime = Duration.between(startInstant, endInstant);
+
+        String output = "";
```

Comment:
Could we use StringBuilder instead of the concatenation?

Reply:
Cleaner this way, I agree. Changes pushed to branch.